content/ru/function-as-a-service.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Функция-как-сервис (FaaS)
+status: Completed
+category: Technology
+tags: ["infrastructure", "", ""]
+---
+
+Функция-как-сервис (Function as a Service, FaaS) — модель облачных вычислений, в которой функции, вызванные событиями, выполняются на сторонней платформе. Ее главный плюс — автоматическое масштабирование, не требующее ручного вмешательства.
+В сущности FaaS позволяет развёртывать отдельные функции, которые активируются в ответ на определенные события, некоторое (короткое) время работают и отключаются. Тем самым гарантируется, что ресурсы не тратятся впустую.
+Модель поддерживает [автоматическое масштабирование](/ru/auto-scaling/), позволяя запускать экземпляр функции по запросу и завершать его после выполнения, подчеркивая stateless-природу.
```

Comment:
However, this [original] "stateless" notice doesn't seem to be really helpful here…
```suggestion
Модель поддерживает [автоматическое масштабирование](/ru/auto-scaling/), позволяя запускать экземпляр функции по запросу и завершать его после выполнения, что соответствует его stateless-природе.
```